wsmaster/integration-tests/pom.xml

```diff
@@ -23,7 +23,14 @@
     <packaging>pom</packaging>
     <name>Integration Tests :: Parent</name>
     <modules>
-        <module>postgresql-tck</module>
         <module>cascade-removal</module>
     </modules>
+    <profiles>
+        <profile>
+            <id>integration</id>
```

Comment:
I think now you can remove integration profile in postgresql-tck module https://github.com/eclipse/che/blob/master/wsmaster/integration-tests/postgresql-tck/pom.xml#L244.